samples-generator/lib/instanceUtilsNext.ts

```diff
@@ -0,0 +1,373 @@
+import { FLOAT32_SIZE_BYTES } from './typeSize';
+import { Attribute } from './attribute';
+import { GltfType, GltfComponentType } from './gltfType';
+const Cesium = require('cesium');
+const Matrix4 = Cesium.Matrix4;
+const Matrix3 = Cesium.Matrix3;
+const Quaternion = Cesium.Quaternion;
+const Cartesian3 = Cesium.Cartesian3;
+const CesiumMath = Cesium.Math;
+
+export namespace InstanceTileUtils {
+    export interface FeatureTableJson {
+        INSTANCES_LENGTH?: number;
+        RTC_CENTER?: [number, number, number];
+        QUANTIZED_VOLUME_SCALE?: [number, number, number];
+        QUANTIZED_VOLUME_OFFSET?: [number, number, number];
+        EAST_NORTH_UP?: boolean;
```

Comment:
This is not being handled right now, but looks like it might need to be: [Refer to Sean's comment on my PR](https://github.com/CesiumGS/swayze/pull/1639#discussion_r403504035)

Reply:
Everything but `createInstancedWithTransform` and `createInstancedOrientation` now use EAST_NORTH_UP, see changes to `instanceUtilsNext.ts`